devops/iam-did-auth-proxy-dev/values.yaml

```diff
@@ -164,6 +164,8 @@ did-auth-proxy-helm:
         listen 0.0.0.0:8080;
         server_name gp4btc-backend-proxy-dev.energyweb.org;
 
+        add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
```

Comment:
In my opinion it is useful, because it applies policy to a lot of endpoints, like `/auth`, `/api-auth` and others. Why it can be dangerous in your opinion?

Reply:
because it can affect subdomains if added later. Been there done that ;) 

IMO it's easier to widen this when necessary than struggle with any potential problems when users have already this cached for one year. I know that clear text HTTP is becoming obsolete. But...

Reply:
> In my opinion it is useful, because it applies policy to a lot of endpoints, like /auth, /api-auth and others.

@JGiter I think that `includeSubDomains` is not necessary to apply the policy to URL paths (like `/auth`, `/api-auth`). I think `includeSubDomains` applies the header to sub domains like `new-sub.gp4btc-backend-proxy-dev.energyweb.org`. 

I think on-balance I agree with Artur that we should not widen to subdomains unless necessary. I don't foresee HTTP for subdomains, but in the very unlikely chance that we do, I can see that this would be a problem due to caching, as @artursudnik said.

Reply:
I get it. Ok, will remove this